src/twilio/twilio_types.bal

```diff
@@ -32,14 +32,81 @@ public type Account record {
 
 # Represents Twilio SMS response.
 # + sid - Unique identifier of the account
+# + dateCreated - The date and time that this resource was created
+# + dateUpdated - The date and time that this resource was last updated
+# + dateSent - The date and time where the outgoing message was sent
+# + accountSid - The unique identifier of the account which sent the message
+# + toNumber - The phone number to which the message was sent
+# + fromNumber - The phone number from which the message was sent
+# + body - The text of the message we want to send
 # + status - Status of the voice call (queued, failed, sent, delivered, undelivered)
+# + direction - The direction of the message (inbound, outbound-api, outbound-call, outbound-reply).
+# + apiVersion - The API version used to process the message.
 # + price - The price amount of the SMS
 # + priceUnit - The price currency
+# + uri - The URI of the resource, relative to https://api.twilio.com
+# + numSegments - The number of segments which make up the complete message.
 public type SmsResponse record {
```

Comment:
IMO, we need to improve the implementation of `mapJsonToSmsResponse` function, in order to map the newly added fields.

Reply:
Added the missing functionality to `mapJsonToSmsResponse` function.